statsmodels/stats/descriptivestats.py

```diff
@@ -310,9 +312,83 @@ def sign_test(self, samp, mu0=0):
     #def sign_test_p(samp,mu0=0):
         #return self.sign_test(samp,mu0)[1]
 
+class DescrStats(object):
+    '''descriptive statistics for data.
+
+    Parameters
+    ----------
+    dataset : ndarray or DataFrame
+        dataset for descriptive statistics.
+    '''
+    def __init__(self, data):
+        '''convert data to pandas dataframe and extract the
+         numeric values
+        '''
+        if isinstance(data, pd.DataFrame) is False:
+            self.data = pd.DataFrame(data)
+        else:
+            self.data = data
+        # select only numerics
+        numerics = ['int16', 'int32', 'int64', 'float16', 'float32', 'float64']
+        self.data = self.data.select_dtypes(include=numerics)
+
+    @OneTimeProperty
+    def obs(self):
+        '''return number of observations'''
+        return self.data.shape[0]
+
+    @OneTimeProperty
+    def mean(self):
+        '''mean of the data'''
+        return np.mean(self.data).values
```

Comment:
I think you could return the pandas Series, i.e. drop values

Reply:
I guess we should also set the `name` for a Series, if it's not automatic

Reply:
> I think you could return the pandas Series, i.e. drop values

How about I use `mean` method of pandas instead of numpy mean ? It returns pandas series.